docs/conformance/cncf/index.md

```
@@ -75,10 +81,12 @@ aicr validate --phase conformance \
   --evidence-dir ./evidence --cncf-submission -f dra -f hpa
 ```
 
-Alternatively, run the evidence collection script directly:
+Alternatively, run the evidence collection script directly. Valid section
+names are `dra`, `gang`, `secure`, `accelerator-metrics`, `service-metrics`,
+`gateway`, `operator`, `hpa`, `cluster-autoscaling`, or `all`:
 ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)